homeassistant/const.py

```diff
@@ -1214,13 +1214,18 @@ class UnitOfPrecipitationDepth(StrEnum):
 class UnitOfSpeed(StrEnum):
     """Speed units."""
 
+    BEAUFORT = "Bft"
     FEET_PER_SECOND = "ft/s"
     METERS_PER_SECOND = "m/s"
     KILOMETERS_PER_HOUR = "km/h"
     KNOTS = "kn"
     MILES_PER_HOUR = "mph"
 
 
+_DEPRECATED_SPEED_BEAUFORT: Final = DeprecatedConstantEnum(
+    UnitOfSpeed.BEAUFORT, "2025.1"
+)
+"""Deprecated: please use UnitOfSpeed.BEAUFORT"""
```

Comment:
Please remove this and update the test instead as explained here: https://github.com/home-assistant/core/pull/106077#discussion_r1469234638

Reply:
I don't get what the test 'test_deprecated_constants' should look like now, in test_const.py. Can you make a proposal?